internal/store/postgres/group_repository.go

```diff
@@ -25,6 +26,17 @@ type GroupRepository struct {
 	dbc *db.Client
 }
 
+type joinGroupMetadata struct {
+	ID        string         `db:"id"`
+	Name      string         `db:"name"`
+	Slug      string         `db:"slug"`
+	OrgId     string         `db:"org_id"`
+	Key       any            `db:"key"`
+	Value     sql.NullString `db:"value"`
+	CreatedAt time.Time      `db:"created_at"`
+	UpdatedAt time.Time      `db:"updated_at"`
+}
+
```

Comment:
not necessarily important but can we move this to group.go?

Reply:
hey, since the scope of this struct will be limited to repository, I think it would be okay to keep it here